README.md

```diff
@@ -22,3 +22,18 @@ Read the [Spec](https://wicg.github.io/aom/spec/) - note, not up-to-date
 [Interactive Slide Deck](http://wicg.github.io/aom/demos/) (for Chrome)
 
 [Tic-tac-toe demo](http://wicg.github.io/aom/demos/tictactoe.html) (for Chrome)
+
+## Notification API for Confirmation of Action
```

Comment:
how about this is a heading 1 (#), or the "Editors" and "Details" below are a heading 3 (###)
so it's clear they're related to the notification API and not to all of AOM?

Reply:
Went with the second option of header 3 (###) for "Editors" and "Details". Thanks!